### ISSUE TITLE
not working for firefox

### ISSUE THREAD

**Issue by markhillard** [DH3ALEJANDRO/brackets-open-in-browser#23]
Firefox is my default browser and it's not opening HTML files (for example).

Here's a snippet from my brackets.json file:

```json
    "open.in.browser.extensionFileDefault": [
        "html",
        "htm",
        "jpg",
        "jpeg",
        "gif",
        "png",
        "ico"
    ],
    "open.in.browser.extensionFileChrome": []
```

Am I missing something or is this actually a bug? The JSON checks out and I'm not getting any console errors in dev tools, so not sure what's going on.

**Comment by DH3ALEJANDRO** [comment]
possibly resolved in version 4.1.2

**Comment by markhillard** [comment]
Upgraded to 4.1.2 and verified that it works. Thanks!
